.github/workflows/ContinuousIntegration.yml

```diff
@@ -0,0 +1,34 @@
+# GitHub workflow to enable continuous integratoin
+name: Continuous Integration
+
+# This workflow is triggered on pushes and pull requests to the repository.
+on: 
+  pull_request: 
+    branches: 'master'
```

Comment:
You're doing this only for pull requests to master?

Elementary does this for pull requests to master but also push on branches:
```
# This workflow is triggered on pushes and pull requests to the repository.
on: 
  push:
    branches: '**'
  pull_request: 
    branches: 'master'
```

Reply:
I copied what I had from the [RECONR](https://github.com/njoy/RECONR) repository. Thanks for a watchful eye.